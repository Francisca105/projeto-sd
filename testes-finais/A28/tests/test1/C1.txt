put <a,1>
put <a,2>
take <a,1>
exit
