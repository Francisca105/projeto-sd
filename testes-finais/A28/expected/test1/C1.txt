ClientMain
> OK

> OK

> OK
<a, 1>

> 